src/http/AuthHttpClient.js

```diff
@@ -216,10 +216,57 @@ class AuthHttpClient {
         return this._instance(request);
     }
 
+    /**
+     * Trigger a token step up notification.
+     *
+     * @param {Object} stepUp - token step up notification payload
+     * @return {Object} response - response to the Api call
+     */
     async triggerStepUpNotification(stepUp) {
-        const type = stepUp.tokenId ? 'tokenStepUp' : 'requestStepUp';
         const req = {
-            [type]: stepUp
+            tokenStepUp: stepUp
+        };
+        const request = {
+            method: 'post',
+            url: `/notify/stepup`,
+            data: req
+        };
+        return this._instance(request);
+    }
+
+    /**
+     * Trigger a balance step up notification.
+     *
+     * @param {Array} accountIds - array of account ids
+     * @return {Object} response - response to the Api call
+     */
+    async triggerBalanceStepUpNotification(accountIds) {
+        const req = {
+            balanceStepUp: {
+                accountIds
```

Comment:
does this work? In the proto, I see account_id , no "s". I thought that meant we need this field to be named accountId in the request-object or else the server's proto-JSON-parser magic wouldn't "notice" this field?

Reply:
In doubt, I replaced it with accountId: accountIds